src/test/e2e/api/admin/feature-archive.e2e.test.ts

```diff
@@ -77,14 +77,13 @@ test('returns three archived toggles', async () => {
 });
 
 test('returns three archived toggles with archivedAt', async () => {
-    expect.assertions(3);
```

Comment:
Why did this have to change?

Reply:
no `archive`, just `archivedAt`

Reply:
I know, that's why I said I had to fix unrelated tests around this, the scheduled-service and maintenance mode. Strange